packages/store/src/utils/utils.ts

```diff
@@ -1,24 +1,28 @@
 /**
- * Returns the type from an action instance.
+ * Returns the type from an action instance/class.
  * @ignore
  */
-export function getActionTypeFromInstance(action: any): string {
+export function getActionTypeFromInstanceOrClass(action: any): string | undefined {
```

Comment:
As stated before, please revert this rename... it is a breaking change.
PS. The existing name is more correct too. It determines the type given an instance, which could be a class based instance or an object literal. This method is not intended to have the class passed to it.

Reply:
It's not passed in action instance all the time. see

https://github.com/ngxs/store/blob/240b84d56e0b4095939327e17bba35c3ebc58a84/packages/store/src/operators/of-action.ts#L78

You may not remember it. But it indeed has class input.

Reply:
Ah, that makes sense, but unfortunately we are stuck with the name for now.
If you want to submit a PR with just only this rename in it than we can include that breaking change when we do the next major version.